dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/modelWriter.py

```diff
@@ -217,19 +218,20 @@ class ModelWriter(ModelWriterBase):
     # @param obj_factory : builder associated to the writer
     # @param mod_name : name of the model
     # @param output_dir : output directory where files should be written
+    # @param package_name: name of the Modelica package containing the model
     # @param init_pb : indicates if the model is an init model
-    def __init__(self, obj_factory, mod_name, output_dir, init_pb = False):
-        ModelWriterBase.__init__(self,mod_name)
+    def __init__(self, obj_factory, mod_name, output_dir, package_name, init_pb = False):
+        ModelWriterBase.__init__(self,mod_name.replace(package_name, ''))
```

Comment:
I propose to put the name without the package name into a class variable to avoid calling this several times

Reply:
I will do it